server/game/cards/03_TWI/events/CornerThePrey.ts

```diff
@@ -0,0 +1,31 @@
+import { EventCard } from '../../../core/card/EventCard';
+import AbilityHelper from '../../../AbilityHelper';
+import { Attack } from '../../../core/attack/Attack';
+
+export default class CornerThePrey extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '6476609909',
+            internalName: 'corner-the-prey',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Attack with a unit. It gets +1/+0 for this attack for each damage on the defender at the start of this attack.',
+            initiateAttack: {
+                attackerLastingEffects: [
+                    {
+                        condition: (attack: Attack) => !attack.target.isBase(),
+                        effect: (attack: Attack) => {
```

Comment:
@AMMayberry1 I am a bit stuck on this card.
This implementation does not work  because effect is `Any` so I do not have access to `attack`. It's probably trivial but with my very limited knowledge ot typescript, I do not see how I can access to defender (or target) damage before the attack starts.
Do you have a few pointers for me?